### ISSUE TITLE
[REQUEST] Remove/Edit Backgrounds (ex. Halloween/Scream Fortress)

### ISSUE THREAD

**Issue by Meladoom** [CriticalFlaw/TF2HUD.Editor#256]
**Describe the feature you're suggesting.**
Flawhud seems to add/change quite a lot of menu pictures, but It'd be nice to change them (ex. Halloween-themed), and also just having the same (custom) image even during Halloween.

Related:
https://github.com/CriticalFlaw/flawhud/issues/119
https://github.com/CriticalFlaw/flawhud/issues/185

**Additional information**
![image](https://github.com/user-attachments/assets/7e22f96a-83f7-4761-8a8a-ec82fd3d3a1f)
![image](https://github.com/user-attachments/assets/17c746af-5b94-43a7-abb8-495f78cdb743)
![image](https://github.com/user-attachments/assets/bd3129a6-9858-4aed-87ec-11cad4be142f)
![image](https://github.com/user-attachments/assets/01ce0432-839c-4627-9a50-f8224772c628)
![image](https://github.com/user-attachments/assets/fa3e303d-d0bc-4ff0-90b5-e7f4d498de3f)

Add any additional details about your suggestion here.
Basically make it easier to change every image here to the ones the user wants
![image](https://github.com/user-attachments/assets/985662b1-4944-47de-b13d-c0c701bb2c23)

Maskillz sketches (second pic is a second window)
![image](https://github.com/user-attachments/assets/2d822a2e-677e-46d1-957f-7efbd951e00f)
![image](https://github.com/user-attachments/assets/99b7d66e-5e42-46f9-9713-3ee0c31794ef)
